### ISSUE TITLE
Failure to open or create volumes with Edge clients in overload situations

### ISSUE THREAD

**Issue by redlicha** [openvstorage/volumedriver#184]
While investigating #182 / #180 : accelio's `listen` backlog is hard-wired to 4. If multiple clients try to connect at the same time (which also happens as part of the "location check"!), some clients might run into errors. Modelling this with a test that creates a volume and then opens it from a number of threads which in turn will then issue `get_volume_uri` requests for a certain amount of time:

```
$ EDGE_STRESS_READERS=32 target/rtchecked/bin/volumedriver_fs_test --volumedriverfs-binary-path=target/rtchecked/bin/volumedriver_fs --gtest_filter=*get_volume_uri_stress* --logfile=/tmp/test.log
Note: Google Test filter = *get_volume_uri_stress*
[==========] Running 1 test from 1 test case.
[----------] Global test environment set-up.
[----------] 1 test from NetworkServerTest
[ RUN      ] NetworkServerTest.get_volume_uri_stress
librdmacm: couldn't read ABI version.
librdmacm: assuming: 4
CMA: unable to get RDMA device list
/home/arne/Projects/openvstorage/dev/volumedriver-core/src/filesystem/test/NetworkServerTest.cpp:1686: Failure
      Expected: 0
To be equal to: ovs_ctx_init(ctx.get(), vname.c_str(), 02)
      Which is: -1
Input/output error
/home/arne/Projects/openvstorage/dev/volumedriver-core/src/filesystem/test/NetworkServerTest.cpp:1686: Failure
      Expected: 0
To be equal to: ovs_ctx_init(ctx.get(), vname.c_str(), 02)
      Which is: -1
Input/output error
/home/arne/Projects/openvstorage/dev/volumedriver-core/src/filesystem/test/NetworkServerTest.cpp:1697: Failure
      Expected: exp_uri
      Which is: "tcp://127.0.0.1:12328"
To be equal to: uri
      Which is: ""
/home/arne/Projects/openvstorage/dev/volumedriver-core/src/filesystem/test/NetworkServerTest.cpp:1697: Failure
      Expected: exp_uri
      Which is: "tcp://127.0.0.1:12328"
To be equal to: uri
      Which is: ""
[...]
[  FAILED  ] NetworkServerTest.get_volume_uri_stress (33017 ms)
[----------] 1 test from NetworkServerTest (33017 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test case ran. (33019 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] NetworkServerTest.get_volume_uri_stress
```

dmesg shows SYN flood messages:
```
[2443407.278775] TCP: request_sock_TCP: Possible SYN flooding on port 12328. Sending cookies.  Check SNMP counters.
```

Modifying accelio's code to use a larger backlog lets this test succeed.
To be investigated whether this is the way to go.

**Comment by redlicha** [comment]
Accelio's backlog was set to "4" to support iWARP in kernel mode: https://github.com/accelio/accelio/commit/0266a035d4184bd79a7bd46bbfa32639a545c094

**Comment by redlicha** [comment]
Addressed by 1.7.2-0ovs1 accelio packages.
